### DISCUSSION TITLE
Home screen widgets

### DISCUSSION THREAD

**Discussion by LeoKlaus** [LeoKlaus/Paperparrot#62] [Feature Requests]
Add widget support to Paperparrot. Widgets could show:
 - Documents in a user-defined saved view
 - All documents in inbox
 - Number of documents for tag/document type/correspondent (gimmick)

**Comment by JigSawFr** [comment]
Nice idea !

**Comment by timbuchwaldt** [comment]
Would love to see it, best case for me would be counters for a few queries (e.g. I have a TODO-Tag, so a count of open TODO tags would be cool, same for Inbox)
